### PR TITLE
Fixes #1454 ValidateConfigOrPanic doesn't seem to be respecting "skiprunmode_validation": true in the config.

### LINKED ISSUE #1454: ValidateConfigOrPanic doesn't seem to be respecting  "skip_runmode_validation": true in the config.
ValidateConfigOrPanic doesn't seem to be respecting  "skip_runmode_validation": true in the config.

### PR DESCRIPTION
**Pull request by tleyden**
https://github.com/couchbase/sync_gateway/issues/1454

### CODE REVIEW

File: src/github.com/couchbase/sync_gateway/rest/config.go [hunk 1]
```diff
@@ -494,7 +494,7 @@ func ParseCommandLine() {
 			config.LogFilePath = logFilePath
 		}
 
-		if skipRunModeValidation != nil {
+		if *skipRunModeValidation == true {
```
Comment (adamcfraser) [inline src/github.com/couchbase/sync_gateway/rest/config.go:497]: This change looks like it's just applying the command line parameter to the config, when a config is used.

It was failing for me when I was trying to run with no command line parameter, and only the config setting.  Is that combination working for you?

Comment (tleyden) [inline src/github.com/couchbase/sync_gateway/rest/config.go:497]: it worked for me, but I'll retest more thoroughly

Comment (tleyden) [inline src/github.com/couchbase/sync_gateway/rest/config.go:497]: Yeah, everything seems to be working for me:  https://gist.github.com/tleyden/a96bd43bee91e7e160d4

Can you post the exact config, binary, and cmd line args you are using into a gist + output?

**Comment by adamcfraser** [comment]
Retested with this branch - it's working for me now.

**Pull request merged**